index.html

```diff
@@ -12,412 +13,495 @@
+                            fill="#6070FF" />
+                    </svg>
+                </div>
+            </a>
+
+            <div class="nav-list">
+                <ul class="nav-list-item">
+                    <li>
+                        <a href="#portfolio">Portfolio</a>
+                        <a href="#about">About</a>
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div onclick="toggle()" class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+
+        </nav>
+        <div class="menupopup" onclick="close1()" id="menupopup">
+            <img onclick="close1()" src="./img/Icon.png" alt="icon-png">
+            <ul>
+                <a href="#portfolio">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
```

Comment:
[Optional] When I open the mobile menu and change the screen size, the mobile menu does not disappear on getting to the desktop version. This might cause a malfunction when a user switches from portrait to landscape screen modes. I suggest that you look into this.

![image](https://user-images.githubusercontent.com/20554583/201728230-6526f25b-6331-4e80-8f76-95cbbb707e89.png)